### PR TITLE
I18n print

### PR DESCRIPTION
**Pull request by zekth**
fixes: https://github.com/qmk/qmk_configurator/issues/368

also i may have some suggestions about translations to make it easier to maintain and contribute. Here is an example: https://github.com/reef-pi/reef-pi/blob/master/docs/internationalization_contributing.md

### CODE REVIEW

File: src/views/Print.vue [hunk 1]
```diff
@@ -117,10 +119,10 @@ export default {
     },
     print() {
       if (this._notes === '') {
-        this._notes = 'My awesome keymap';
+        this._notes = this.$t('message.layer.empty');
```
Comment (noroadsleft) [inline src/views/Print.vue]: Should be `message.notes.empty` here.

```suggestion
        this._notes = this.$t('message.notes.empty');
```

**Comment by noroadsleft** [comment]
Firefox seems to use the `placeholder` attribute values for printing if the user doesn't set their own. I'm okay with this though.

image

**Comment by zekth** [comment]
> Firefox seems to use the `placeholder` attribute values for printing if the user doesn't set their own. I'm okay with this though.
> 
> image

This seems unrelated with my changes but didn't know FF uses placeholder value if empty

**Comment by yanfali** [comment]
@zekth if you print, those go away and are replaced with stock text.

**Comment by yanfali** [comment]
I think this is great. My plan was however to leverage ES6 imports and simply put this into it's own file.  And include it into index.js much like potato.js. This is really it's own separate application that leverages the same data store. Can we create a new file with these keys called `print.js`, import that and then include it using `...print`. Let's also namespace this into it's only subtree like potato:

```JSON
export default {
  print: {
     ...rest of keys here
  }
}
```

I'm about to add new part of the application to test keypresses, and I was planning on localizing it using this scheme. Let me know what you think. The CSV idea is interesting, I will think about it.

**Comment by zekth** [comment]
Internationalization is really not a fun part for any developper. Using CSV is pretty straight forward and easy to read / edit. Using Js fles could be long. If you want i can address that.

Also having multiple files for namespaces has no real benefits as there is not so much entries. Or if you prefer `index.js` would just refers all the namespaces, so we would need to add something like `common.js`

**Comment by yanfali** [comment]
This looks good. Thank you. I will test it today.

**Pull request merged**